### PR TITLE
Make the code for adding panels DRY

### PR DESCRIPTION
**Pull request by davidmosterd**
username: davidmosterd
issue: #307

### CODE REVIEW

File: inc/customizer.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ function twentyseventeen_customize_register( $wp_customize ) {
 	for( $i = 1; $i <= 4; $i++ ) {
 
 		$wp_customize->add_section( 'panel_' . $i, array(
-			'title'           => __( sprtinf( 'Panel %s', $i ), 'twentyseventeen' ),
+			'title'           => __( sprintf( 'Panel %s', $i ), 'twentyseventeen' ),
```
Comment (enodekciw) [inline inc/customizer.php]: This is not the way it should be :)
You first have to translate the string, then pass in the variables. Like this:
`sprintf( __( 'Panel %s', 'twentyseventeen' ), $i );`

Comment (davidmosterd) [reply inc/customizer.php]: You are right, very sorry. I will fix this.

**Comment by davidakennedy** [comment]
@davidmosterd This is a cool idea, and I like the less code. 😄 

But in the case of a default theme, I don't mind doing it the old-fashioned way so anyone learning from the theme understands the Customizer a bit better.

Thank you for the pull request!

**Comment by davidmosterd** [comment]
@davidakennedy Ok, default themes are good for this I agree. I have also considered the option of making this a function (e.g. `twenteseventeen_customize_add_panel`)  and calling it four times with our without parameters. It will not introduce a thing like a for-loop. But I will show people not to repeat code, which is also good? I am new to default-theme development so maybe this is still not the old-fashioned way but wanted to suggest it anyway.

**Comment by celloexpressions** [comment]
I completely disagree with the claim that this is too complex for a default theme. Or even that not doing this is the "old fashioned" way. Using an iterator to loop through and create the multiple instances, still via the vanilla customizer API is the correct way to do this. This is how core would do it if several numbers objects had to be repetitively created. The PR, in my opinion, is exactly how the theme should do this (also eventually making the corrections in #348 to remove the excessive section hierarchy, which will further simplify).

On the contrary, the current approach of copy/pasting the same thing four times adds a lot of excessive complexity, making it look like there's more there than there is because the code is so much longer. This also doesn't need to be moved to a function; a basic for loop in the customize_register callback is the clearest approach.

**Comment by davidmosterd** [comment]
As a programmer: I agree with @celloexpressions (hence the pull request), as we should not learn people the bad practice of repeating yourself and increase the risk of bugs. As a contributor: I'm new to WordPress theme-contribution so I did not/ do not want to sell my opinion too much as I am still learning the culture :)

**Pull request closed without merging**